packages/core/src/domain/session/sessionStore.ts

```diff
@@ -18,8 +19,10 @@ export interface SessionStore {
   restartSession: () => void
   renewObservable: Observable<void>
   expireObservable: Observable<void>
+  forceReplayObservable: Observable<void>
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Shouldn't we do a more generic observable/setter. We only have the `forceReplay` setting, but maybe we'll get more settings like this one. We could have `settingsObservable` that notifies sessionManager with the type of setting.

Reply:
Agree, but that could be done later, when we have new use case. Reviews from the previous PR advised to keep exposed methods and logic restricted to forced replay.